### ISSUE TITLE
No url found for submodule path 'vendor/github.com/bitly/go-simplejson' in .gitmodules

### ISSUE THREAD

**Issue by svanas** [adshao/go-binance#6]
`go get github.com/adshao/go-binance`

..is giving me this error:

```
git submodule update --init --recursive
fatal: No url found for submodule path 'vendor/github.com/bitly/go-simplejson' in .gitmodules
package github.com/adshao/go-binance: exit status 128
```

**Comment by adshao** [comment]
Hi @svanas , could you please refer to https://github.com/adshao/go-binance/issues/2#issuecomment-357835448 for more info.

**Comment by adshao** [comment]
resolved in #2
